File: 119-get-row/Solution.java
import java.util.*;

class Solution119 {
    public List<Integer> P0119_getRow(int rowIndex) {

        List<List<Integer>> mainList = new ArrayList<>();
        mainList.add(new ArrayList<>(Arrays.asList(1)));

        if (rowIndex == 0){
            return mainList.get(0);
        }

        for (int i=1; i<=rowIndex; i++){
            List<Integer> subList = new ArrayList<>(Arrays.asList(1));

            for (int j=1; j<i; j++){
                subList.add(mainList.get(i-1).get(j-1) + mainList.get(i-1).get(j));
            }

            subList.add(1);
            mainList.add(subList);
        }
        return mainList.get(rowIndex);
    }
    static void main(String[] args) {
        Solution119 sol = new Solution119();
        System.out.println(sol.P0119_getRow(0));
    }
}

File: 119-get-row/solution.py
from typing import List
class Solution1122:
    def getRow(self, rowIndex: int) -> List[int]:

        mainList = [1]
        if rowIndex == 0:
            return mainList
        
        for i in range (1, rowIndex + 1):
            subList = [1]
            for j in range(1, i):
                subList.append(mainList[i-1][j-1]+mainList[i-1][j])

            subList.append(1)
            mainList.append(subList)

        return mainList[rowIndex]
                
print(Solution1122().getRow(3))